doc/source/User_guide/extensions.rst

```diff
@@ -28,6 +28,12 @@ Project extension apply to all extensions that are applicable for all AEDT appli
 
             Import a Nastran or STL file in any 3D modeler application.
 
+   .. grid-item-card:: Configure Layout
+            :link: pyaedt_extensions_doc/project/advanced_fields_calculator
+            :link-type: doc
+
+        Advanced Fields Calculator.
```

Comment:
I'm assuming that you want "Advanced Fields Calculator" treated as a proper name? If so, you will want to add three words in this form to a line in Vale's accept.txt file. Be sure to be consistent. Making this description more comprehensive, like the earlier ones. For example: "Learn how to use the Advanced Fields Calculator" to configure the layout." (I need somehow a way to understand the "Configure layout" heading in a more detailed description.)